nix/sources.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
lets add iohk-nix (see iohk-ops for how). We can put genesis files/hashes and any other env specific data in there that we can use to key off of. Granted, we're using iohk-ops here so it's not 100% needed but could be useful for end users wanting to run a faucet without iohk-ops. See cardanoLib in there. We'd want to add a jormungandr_testnet attribute with env specific stuff in there.